.prettierrc

```diff
@@ -1,7 +1,14 @@
 {
-  "tabWidth": 2,
-  "useTabs": false,
```

Comment:
Do we want to retain this?

Reply:
I'll get rid of that I think

Reply:
We weren't using prettier before to lint, so I don't think it matters. For now this is just to fix the imports

Reply:
We can always revisit it later.